web/store/message.ts

```diff
@@ -6,6 +6,7 @@ import { local } from './data/storage'
 import { subscribe } from './socket'
 import { toastStore } from './toasts'
 import { userStore } from './user'
+import { setSwipe } from '../pages/Chat/ChatDetail'
```

Comment:
A store shouldn't import from a component. This function should be passed as a parameter to the function instead.